src/02-data-types.js

```diff
@@ -22,7 +24,9 @@ function getLengthOfString(str) {}
  *  convertToNumber("111");
  *  //> 111
  */
-function convertToNumber(val) {}
+function convertToNumber(val) {
+  return parseInt(val);
```

Comment:
Rememver we can also use ` Number(val) ` in this instance.  ` ParseInt() ` works but will give a different answer if we pass in a decimal like ` 1.5 `;